qriollo.rb

```diff
@@ -0,0 +1,29 @@
+class Qriollo < Formula
+  desc "An impure functional language, based on Rioplatense Spanish."
+  homepage "https://qriollo.github.io"
+  url "https://qriollo.github.io/Qriollo-0.91.tar.gz"
+  sha256 "c8357af8254a082d8e4da1de1bbf13bee27cfde8adb31ea0a5a0966bfbb7b28d"
+  head "https://github.com/qriollo/qriollo.git"
+
+  depends_on "ghc" => :build
+
+  def install
+    system "make"
+    bin.install "qr"
+    (lib/"chamuyo").install "Chamuyo.q"
+  end
+
+  def caveats
+    "The standard module \"Chamuyo.q\" has been placed in #{lib/"chamuyo"}."
```

Comment:
If you wrap `def caveats` with `<<-EOS.undent` you won't need to escape the double quotes.